### DISCUSSION TITLE
什么时候能上手机客户端呢

### DISCUSSION THREAD

**Discussion by ethenlong** [enfein/mieru#22] [General]
rt

**Comment by enfein** [comment]
安卓这边要取决于 SagerNet 出新版本的进度。其实 SagerNet mieru 插件在 GitHub 已经可以下载了。

https://github.com/SagerNet/SagerNet/releases/tag/mieru-plugin-1.6.0

SagerNet 用 0.8.1-rc02 版本就可以。

苹果这边暂时还没有和任何开发者合作，也没有自建 APP 的计划。

**Reply by V70024** [reply]
@enfein
If possible, send me a message on Telegram, I have a suggestion 
[t.me/v70024](https://t.me/v70024)

**Comment by enfein** [comment]
考虑到 SagerNet 这个项目已死，mieru 2 不会继续支持 SagerNet。之后会开发官方的安卓手机客户端。

**Reply by Jiraiya8** [reply]
方便透露一下开发进度么？自己开发官方手机客户端的话，还是有很多东西需要处理的，比如分流规则。目前看来想要用上手机客户端还很漫长

**Reply by enfein** [reply]
正在学习移动开发，希望 2024 年底可以做出一个粗糙可用的 app 出来。

**Reply by ayanami-desu** [reply]
有其他类似sargenet的项目，比如nekoray。感觉没必要造轮子

**Reply by PharosVip** [reply]
[Please check out this advice and Routen Proxy Client](https://github.com/enfein/mieru/discussions/60)
